src/modules/Users.pm

```diff
@@ -1519,7 +1385,6 @@ sub ReadAvailablePlugins {
 
     # update internal keys with the values from plugins
 
-    my @user_internals	= @{UsersLDAP->GetUserInternal ()};
```

Comment:
Here the declaration is removed, but not the use of @user_internals in the following lines. Not sure what to do, honestly, if remove those lines or re-declare @user_internals to be an empty hash (!!??)

@jreidinger, @joseivanlopez could you help here?

Reply:
it should be redeclared with empty hash I think